docs/admin/install.rst

```diff
@@ -83,6 +83,8 @@ of them in :file:`requirements-optional.txt`.
     https://python-zeep.readthedocs.io/
 ``aeidon`` (optional for :ref:`subtitles`)
     https://pypi.org/project/aeidon/
+``ruamel.yaml`` (optional for :ref:`yaml`)
+    https://pypi.org/project/ruamel.yaml/
```

Comment:
Thanks, but it should actually replace PyYAML above. This has changed in translate-toolkit we use for parsing the files and where new version was released just after release of Weblate 3.7, so it is not covered by docs so far...